### ISSUE TITLE
SyntaxError: Invalid regular expression: /^+(.*)$/: Nothing to repeat

### ISSUE THREAD

**Issue by sakunyo** [inuscript/historykana#30]
When history array has chars for Regex with string take be error.
Should be escape Regex value it. Or Should be ignored Regex chars. :cry

```js
var histKana = require("historykana");

var ok = histKana([ "ｙ", "や", "やｍ", "やま", "山" ]);
console.assert(ok === "やま");

var ng = histKana([ "ｙ", "や", "+やｍ", "+やま", "+山" ]); // History string has Regex meta character.


// Error message.
// 
//  var reg = new RegExp("^" + left + "(.*)" + right + "$");
//            ^
//  SyntaxError: Invalid regular expression: /^+(.*)$/: Nothing to repeat
```

**Comment by terrierscript** [comment]
Thank you for bug report!

**Comment by terrierscript** [comment]
I fix this #31 and published 1.0.4
